orchestrator/utils/search_query.py

```diff
@@ -97,6 +97,11 @@ def lex(self) -> Iterable[Lexeme]:
         yield Token.END,
 
 
+def camel_to_snake(s: str) -> str:
```

Comment:
I think we now have this function in multiple locations

Reply:
Moved to utils.helpers